apps/chat/src/components/Chat/ChatMessage/ChatMessageContent.tsx

```diff
@@ -440,7 +474,14 @@ export const ChatMessageContent = ({
                           />
                         </div>
                       }
-                      selectedFilesIds={newEditableAttachmentsIds}
+                      selectedFilesIds={newEditableAttachments.map((f) =>
+                        f.contentType === FOLDER_ATTACHMENT_CONTENT_TYPE
+                          ? ApiUtils.decodeApiUrl(f.id).replace(
+                              new RegExp('^metadata/'),
+                              '',
+                            ) + '/'
+                          : f.id,
+                      )}
```

Comment:
Maybe for the readability it is better to extract all this lines into the variable?